test/core/templates/gif-module-test.js

```diff
@@ -0,0 +1,33 @@
tNkKjokLHCHTBX/zGlA/F43arMd4b9qZ/kOMDGvPe66veR1ZyRiNxpa/xvIR7bfXc1lImzS7AiQOIGrkUmU4QPDSSiscZgJJMDG+RqfpCQQ9LihA+JJBJ4q2e4MSVlPbieRA599GCk4XbTcsLyJ6CzAgMM6OzqGVbiPSXyAs+l/AmVVEJLCnfcsQsu5GtyhgNuyEqmn9jr2sEszMOQwPz0ZzLWPWDEE48opFP5p+wHgd/yOEC/AqLhK3+JyihWUD0lbSQa0YCEBCW4jApGQyYCPAEHAFHABKDhHatB4B/G4oQSyupF7XtIJKQghTqQ4IUwhOEyIAjBSLQkEMwjYAdB2B88jDA2WTnhOP/asBFI1MGFMUxiDEcwjiauBIccEIAAOpgAHj7IFH/I4m1wsA3KsKEJfrqIFKKBREuEARo34MQWgHADL2zhBkscwTJSMotA4PABVJwfCJfwBxFl8Q8jUAATBNmGICGOfQ+RwjhI4IVUPOIQWcjCHOYQyUFAYwyD4AQnwtBGEshxGZAwCQ41yME85iEPaBAGM7bBylZCBxy1QIIHTEDLNjDhlricReYUEo0RkIATlHxBAIZJTGJSI5KSfIQyX1iHaIyDJKPEYx7pl0oxFakG1UCNEgbATSSgAAUmKIA4x1kA3/1vIZB4ITCFWcx2BkCYL6AGEAZxCC/A8IIjieY06Zf/h0woo2xsGIVAKcGLFnBzAEdYwAI0EM4CmIiWTGhDIHZZkBX20gs3gAU73enOF2ThmIewxAvxKRJ97jMBDygHoXoAPZSRwQQoWMBB4TEEhprAA/BYgDcdEVE3hC0SK0zBDbYAC2pw9KjGhAUcm5nP5Ulzn+1IywpGEa8eZIUMTdDAQQdQ028WYAjcXAAKHFGAQiJEAWitwip0gAEM8OCtcOVBAOTazrhSQwciEMEi6FFSp540Ae3QW8oGuwIkbPUI30TBVw/K0AKwUAopCOVAHLGByuaCD4r4wTM2+wMQeDauxYyrZxWRi1wYYxh9PcFTqfiAlBIqK6OQlzWsuoIN/xzhoGL1Kli7aYIVKjIFCrBBBCLAhcpeIg460IEiNmsAAzyjsz/orGen+wMM/KAX0X2GIvgQhXlgIwStKIF4pcGFjcwCflGcJi1GlbJR3OEKb3gDA3rQAzJ44KCI1S03jzDLFPgXrY64xDwGHAUk1MIea0jwBWwg3F5wIQIXWEMvJtwLEPRiDReIQC+GawM/2OLDw0CELBCAgBCEIhQlkIY0MtIG+K2WfgIIRDXIIJip3kG+LYXeOO67Xw94dQFH4C84J6iAKmxAwFEoMBJa4IcEY/gCC4YAgxdM4eEOdw3ClfIFPEyDIdBAFmAGs4lRrGIIXMQNLs4jOgRDtqzcOP/HLWUDj/fr1cR+0wRnk4EPuLvkFrQAGHYAgAsGTegSuADFLgCAig3tAi5AgAvSEHSiXVACAFgaAHrItB7CXOJQYKMEXNhDRdAciCiWkn62W4Hi3IzjlsprGVrlZk7BeVMP+BgF1oiFGSSgiuTaww9+oAGgJW2IQRvi2Ci2NAQkLQ0ILNvSiRZ0pS+tB3NoOtOyCIG2DRFqi7BhexukHyBS3Wb3thp6ZECHVmk6hAWYwBEmGAA8aMrQGwABCLBQhAEusOxoh0IW6wj4OsxBcIKXWNsHP7HCtR0CEushHBAPR8EJjm1Oh0IaEfD29uSXAAE4r2FksIYRzg09WWkAHhr/uHO85w0PHzvCEYsoQxkuaw8bQGDQlg5BtSc+8YM3vNML/3nDZWEOiPN84thGwMW/IOqKzAUdzGveYIkw8hyvoAZMIKfWtU5WI28gCi0IdCiGIQtsH53g6lDH2c0R5raDedNEt/bOeQ53RHyavBBoukRGF/WPN4zq57561re+9ZcX4OWLmMctgAGAf6+94GlXR8R57vaya5rom860xCWOdD0gIgTj5QLTKdKBGjAPHTWoMRFygCmBLvDqtKDFODpBAAYsI/a0mIUbdu+GNijgyJdAAg3sUIK4Pz7tEYf4OoxecGxf+/KW3/TRK65tbGDDENL4QsYlEqpQEUlPbEhG/wVcjak7fR9ODuhYkuAkJzf8Hh+XqMUQ7BCKx0Ne8smffPPl/vzMX57znTdiDIdiovcFTictiBJbrlY2myIoRhJGBgEBJXBiCDB8jWd/aJd/yTdwa9d/nBYO7EB3cCcLiEB2CGBozWYR1kMGtQc9BIA3hGE95yQQWAYB0mAIJBZ3yWd/+KeBynd2O3dtYYYIO0hxQqh2siBeAPAFF+B0eDNjbNBE6daAAtEB1UAL6TcQNzdo2BAKCCALw1BwAYeBPeiDzGd/FUeCYAZxIWiEmmZ0YlZpTUh6CPiADpQQNkBphhAK2mZ8BHeGa1eGZrh5ReiGQ7hp+td8mhdx6iALF/9nAxfRfS9CUQchgV74hWB2dIB4dII4iBoogm2HiInIf/m3aaEgCJCIEgCADQC3doXIc5HniZ5Id273cKNohPmXdqDHhCiBDQgQhjy3fOEgjAB4f7J4jEdXi6+IixpodxBggCbhAqx4dsR4i8h3jNjIfMp4i2+ogcMQAtmXiiRhCCEAjOZAjNUIiNm4jokId7Y4it2Yf2RXadA4jiEwcNj4h+y4j4b4joAYjzvoiNlnEjg4DOqQjvuYkD7IgXJXdjsIgAA5eeswASVQjyNRkAqZkbKoiA4ZcewAkYuYfwQ3AZhgkSLhAr+okSqpgQzpfMn3kYrojxJJkqRgEgBQjiuJmZM/2HwdeYbP54PWRpMm8QUTgAjnqJMqyZDW1pPsaA7qIJQlQZTVhpBIuY5KmXkKaW2tgAkpIQgTMHec2IlVuYwhqYGRp3ZHhwiCoBKCEApGeXxjOYidJ5PJF3lrp5Yr8ZU8GJdmOJeD6BN6CZd8KZIFhyGBGYgr+RAEJzuC0ApvWZgdsZgbERAAIfkEBQAA/wAsAAAAAAEAAQAACAQA/wUEACH5BAUAAP8ALAAAAAABAAEAAAgEAP8FBAA7';
```

Comment:
Please store this in a separate file like `IS-QR.js`. Let's keep the template code super clean.

Reply:
done